dataikuapi/dss/admin.py

```diff
@@ -1143,22 +1143,103 @@ def start(self):
             raise Exception('Cluster operation failed : %s' % (json.dumps(resp.get('messages', {}).get('messages', {}))))
         return resp
 
-    def stop(self, terminate=True):
+    def stop(self, terminate=True, force_stop=False):
         """
         Stops or detaches the cluster
 
         This operation is only valid for a managed cluster.
-        :param boolean terminate: whether to delete the cluster after stopping it
+
+        :param terminate: whether to delete the cluster after stopping it
```

Comment:
We usually put the type in the same line as the param, is there a reason to change ?

Reply:
Hmm, it doesn't seem like it